WcaOnRails/app/views/competitions/_registration_requirements.html.erb

```diff
@@ -20,9 +20,11 @@
     <%= t('competitions.competition_info.register_below_html') %>
     <br>
   <% end %>
-<% elsif @competition.website %>
-  <%= t('competitions.competition_info.register_link_html', here: link_to(t('common.here'), @competition.website, target: "_blank")) %>
-  <br>
+<% else %>
+  <% unless @competition.external_registration_page.blank? %>
+    <%= t('competitions.competition_info.register_link_html', here: link_to(t('common.here'), @competition.external_registration_page, target: "_blank")) %>
+    <br>
+  <% end %>
```

Comment:
Wouldn't it match the existing pattern a little more to combine lines 23 and 24?

```ruby
<% elif competition.external_registration_page %>
   <%= t('competitions....
<% end %>
````

Reply:
Yes, sure! Although I will use `!@competition.external_registration_page.blank?` instead, as `NULL` strings seem to get converted to `""`, and because of this if I use what you proposed the automatic sentence is always displayed.

Reply:
Thanks for the tip, updated!